rpm/xcache.spec

```diff
@@ -202,6 +205,15 @@ mkdir -p %{buildroot}%{_sysconfdir}/grid-security/xrd
 %config(noreplace) %{_sysconfdir}/xrootd/config.d/95-cms-xcache-logging.cfg
 
 %changelog
+* Mon Aug 5 2019 Edgar Fajardo <emfajard@ucsd.edu> - 1.1.0-0.4
+- Adding RucioN2N config for ATLAS Xcache (SOFTWARE-3784)
```

Comment:
Since we haven't tagged a new RC, you can just add this changelog entry to the one below and keep the release version 0.3

Reply:
Well I can add both things into a single RC.